book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+(er-ethics-open-source-governance)=
+# Ethical Considerations when Choosing an Open Source Governance Model
+
+(er-ethics-open-source-governance-summary)=
+## Summary
+
+Open source projects are rooted in ethical principles. 
+Sharing the code so that anyone can use it, welcoming contributors and creating communities around specific projects are the core of open source. 
+But open source communities also have the responsibility to ensure they are a safe, inclusive environment, understand who might be affected by their projects or check that their tools are not being misused. 
+Therefore, Open Source Governance Models should embed those ethical considerations within their structure, but the framework to do so is far from well-established. 
+This chapter discusses some of the key points to take into consideration when designing and implementing ethical guidelines in governance models.
+
+
```

Comment:
```suggestion

```{note}
**What does ethics in governance mean?**

*Ethical behaviour in governance is defined as the way in which stakeholders try to manage collective action from the perspective, and in the interest, of the majority, thus avoiding damaging behaviours, and through a better control of the power and responsibilities of the leaders. In the area of governance, therefore, ethics aims at raising awareness of the others' rights and common needs, by imposing some principles of minimum requirement. 
Source: Dessain, V., Meier, O., & Salas, V. (2011). Corporate Governance and Ethics: [Shareholder Reality, Social Responsibility or Institutional Necessity? Management, 11(2), 65–79](from https://www.cairn.info/revue-management-2008-2-page-65.htm).
```
```